src/libgeodecomp/io/logger.h

```diff
@@ -18,6 +18,10 @@ namespace LibGeoDecomp {
  * happens at compile time, thus no runtime overhead is incurred for
  * filtered messages.
  */
+#ifdef ERROR
```

Comment:
hmmm, good question. It was causing some problems for me before :/ Was `ERROR` used before in your logging code? Originally I introduced this because some windows header defines `ERROR` which got in the way.